alacritty/src/display/content.rs

```diff
@@ -233,19 +237,21 @@ impl RenderableCell {
             let config_fg = colors.search.matches.foreground;
             let config_bg = colors.search.matches.background;
             Self::compute_cell_rgb(&mut fg_rgb, &mut bg_rgb, &mut bg_alpha, config_fg, config_bg);
-
-            is_match = true;
         }
 
+        // Convert cell point to viewport position.
+        let cell_point = cell.point;
+        let line = (cell_point.line + content.terminal_content.display_offset as i32).0 as usize;
+        let point = Point::new(line, cell_point.column);
```

Comment:
Can we make this into a helper function? I've read it twice already in the first 30 lines of this patch.

Reply:
I believe this exists twice in general. I've considered it but it really has no place in alacritty_terminal. And there's no other great spot that is shared between the two places where it is used.